test-framework/emulator_backend.go

```diff
@@ -431,3 +451,30 @@ func newInterpreter() (*interpreter.Interpreter, error) {
 		}),
 	)
 }
+
+func (e *EmulatorBackend) replaceImports(code string) string {
+	if e.configurations == nil {
+		return code
+	}
+
+	program, err := parser.ParseProgram(code, nil)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, importDeclaration := range program.ImportDeclarations() {
+		location, ok := importDeclaration.Location.(common.StringLocation)
+		if !ok {
+			continue
+		}
+
+		address := e.configurations.Addresses[location.String()]
+
+		locationStr := fmt.Sprintf(`"%s"`, location)
+		addressStr := fmt.Sprintf("0x%s", address)
+
+		code = strings.Replace(code, locationStr, addressStr, 1)
```

Comment:
Maybe use the offsets in the position info to replace the code for this particular import declaration only. The code may contain other string literals matching `locationStr` which should not be replaced.

Reply:
Good idea. added with 22580ffec638e88bfb4c5d8e2edcf525cda4e4a9